pyreal/transformers/base.py

```diff
@@ -203,6 +203,19 @@ def fit_transform(self, x, **fit_params):
         self.fit(x, **fit_params)
         return self.data_transform(x)
 
+    def inverse_transform(self, x_new):
```

Comment:
Currently, we have `transform` and `data_transform`, which do the same thing (`transform` calls `data_transform` in the base Transformer class, and all Transformer classes just implement `data_transform`). We decided to do this for clarity. I think it would be good to keep the inverse transform consistent - so `Transformer` will have an abstract `inverse_data_transform` method, and an `inverse_transform` method that simply calls `inverse_data_transform`. Then, all other Transformer classes will implement `inverse_data_transform`.